seed/authentication.py

```diff
@@ -16,17 +18,28 @@ class SEEDAuthentication(authentication.BaseAuthentication):
     """
 
     def authenticate(self, request):
-        auth_header = request.META.get('HTTP_AUTHORIZATION')
+        auth_header = request.META.get('Authorization')
+
+        if not auth_header:
+            auth_header = request.META.get('HTTP_AUTHORIZATION')
 
         if not auth_header:
             return None
 
         try:
-            from seed.landing.models import SEEDUser as User
+            if not auth_header.startswith('Basic'):
+                raise exceptions.AuthenticationFailed(
+                    "Only Basic HTTP_AUTHORIZATION is supported"
+                )
+
+            auth_header = auth_header.split()[1]
+            auth_header = base64.urlsafe_b64decode(auth_header)
```

Comment:
this was a previous pull request and we never approved it because this broke the site. I can't recall why at the moment, but it seemed that base64'ing the auth header was not needed because we are running the site over https. If we base64 the auth header, then many of the tests broke.